src/app/components/footer/footer.component.html

```diff
@@ -1 +1,5 @@
-<p>footer works!</p>
+<footer class="footer">
+    Copyright &copy;
+    <a class="footer-link" href="javascript:void(0)">Videocourses</a>. All rights
```

Comment:
Using _href="javascript:void(0)"_ is a little bit old-fashioned
You can change it to more modern version: _href="" (click)="$event.preventDefault()"_
Or just use button here)
You can read about **$event** in Angular here [Event Binding](https://angular.io/guide/event-binding)

Reply:
Ok, done